+adi/+internal/AD5760.m

```diff
@@ -0,0 +1,92 @@
+classdef (Abstract) AD5760 < adi.common.Attribute & ...
+        adi.common.Channel & ...
+        adi.common.DebugAttribute & ...
+        matlabshared.libiio.base
+    properties
+        PowerDown = 1
```

Comment:
Can you convert this to a logical: properties(Logical)

Reply:
For PowerDownMode, I thought rather than mapping false and true to '6kohm_to_gnd' and 'three_state', indexes 0 and 1 would be a better choice.